.github/workflows/dhis2-verify-app.yml

```diff
@@ -114,7 +114,7 @@ jobs:
                   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
                   CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
                   CYPRESS_dhis2BaseUrl: https://debug.dhis2.org/dev
-                  CYPRESS_dhis2ApiVersion: 39
+                  CYPRESS_dhis2ApiVersion: 40
```

Comment:
Do we actually want this, should we not run the fixtures against `2.39/dev`?

Reply:
Hmm I think 2.40 should be fine. It should have the same endpoints (- it does afaik) and once we release these should point to 2.40 anyway. I wouldn't mind changing it, do you see an advantage in setting them back to 39?

Reply:
Should be fine, but it can be unstable. `2.39dev` should really be what we run these tests towards, since that's the version we are targeting currently.